Python/Product/TestAdapter.Executor/Pytest/PytestTestDiscoverer.cs

```diff
@@ -150,11 +150,6 @@ ITestCaseDiscoverySink discoverySink
             arguments.Add("--");
             arguments.Add("--cache-clear");
             arguments.Add(String.Format("--rootdir={0}", projSettings.ProjectHome));
-            if (string.IsNullOrEmpty(projSettings.UnitTestRootDir)) {
-                arguments.Add(String.Format("{0}", projSettings.ProjectHome));
-            } else {
-                arguments.Add(String.Format("{0}\\{1}", projSettings.ProjectHome, projSettings.UnitTestRootDir));
-            }
```

Comment:
It doesn't matter much, but now that there's no conditional here, the whole thing can become a single array-new expression for better readability:
```cs
public string[] GetArguments(...) =>
    new[] { DiscoveryAdapterPath, "discover", "pytest", ... };
```

Reply:
@int19h Hi Pavel, thanks for the comments. It makes sense to me:) I just modified the code based on your comments, please let me know if you notice anything else. Thanks!